lib/modules/versioning/aws-eks-addon/index.ts

```diff
@@ -0,0 +1,22 @@
+import { RegExpVersioningApi } from '../regex';
+import type { VersioningApi } from '../types';
+
+export const id = 'aws-eks-addon';
+export const displayName = 'aws-eks-addon';
+
+export const urls = [];
+
```

Comment:
```suggestion
```

no need to export empty urls, they are optional.

Reply:
Added back. Explained below